Formula/yara.rb

```diff
@@ -12,17 +12,27 @@ class Yara < Formula
     sha256 "deba43f8ebc4d52d0d8a17febdebf8cc5b0de6c9f81f28e7d6a03aea2a87abe7" => :el_capitan
   end
 
-  depends_on "libtool" => :build
+  option "with-enable-dotnet"
```

Comment:
Given that it doesn't require an extra dependency, please make it default (with no option)